boot/zephyr/Kconfig

```diff
@@ -360,8 +361,12 @@ config UPDATEABLE_IMAGE_NUMBER
 	help
 	  Enables support of multi image update.
 
+choice
```

Comment:
If this is a choice, we need an option to choose neither of these. There are existing uses of MCUboot where rolling back is a required feature, and that still needs to be possible.

Reply:
I marked this choice as `optional` , so it can be completely switched off.
FYI: HW rollback protection allows for rolling back the firmware to some extent
(the security counter doesn't need to be increased with each sw release).